src/components/Text/Text.tsx

```diff
@@ -0,0 +1,23 @@
+import {Text as RNText, TextProps as TextProperties, TextStyle} from 'react-native'
+import {metrics} from '../../helpers/metrics'
+import styled from 'styled-components/native'
+
+type TextProps = {
+  fontSize?: TextStyle['fontSize']
+  color?: string
+  fontWeight?: TextStyle['fontWeight']
+} & TextProperties
+
+export const Text = styled(RNText)<TextProps>(props => ({
+  fontSize: props?.fontSize || metrics.span,
+  color: props?.color || props?.theme?.colors?.black,
+  fontFamily: props?.theme?.fonts?.regular,
+}))
+
+export const TextBold = styled(Text)<TextProps>(props => ({
+  fontFamily: props?.theme?.fonts?.bold,
+}))
+
+export const TextItalic = styled(Text)<TextProps>(() => ({
+  fontStyle: 'italic',
```

Comment:
Should we have `fontFamily` for Italic here?

Reply:
The default fonts of Android do not support Italic so that is why I have to use `fontStyle`

Reply:
Yeah, but in iOS, I see we have an Italic font, so shouldn't we need to add that?
<img width="455" alt="Screenshot 2023-08-04 at 14 18 54" src="https://github.com/saigontechnology/rn-base-component/assets/114563576/0a9467f4-9c50-451d-800a-f063c63f4e87">

Reply:
Let's discuss in the team meeting